gemm/include/MatrixMult.h

```diff
@@ -0,0 +1,15 @@
+# pragma once
+
+#include "Matrix.h"
+
+class MatrixMult {
+ public:
+    MatrixMult(std::unique_ptr<const Matrix> A, std::unique_ptr<const Matrix> B)
```

Comment:
I know we worked through several permutations of this. I thought we had settled on passing in output C to avoid memory allocation being part of the benchmark.

Reply:
I've updated the branch to include `_setup`, `_run` and `_teardown` methods so that each can be benchmarked individually.